workflow/rocoto/gfs_tasks.py

```diff
@@ -533,13 +533,13 @@ def aeroanlgenb(self):
     def aeroanlinit(self):
 
         deps = []
-        dep_dict = {'type': 'task', 'name': f'{self.cdump}aeroanlgenb', 'offset': f"-{timedelta_to_HMS(self._base['cycle_interval'])}"}
+        dep_dict = {'type': 'task', 'name': 'gdasaeroanlgenb', 'offset': f"-{timedelta_to_HMS(self._base['cycle_interval'])}"}
```

Comment:
This should probably be built off of `run` even if it is only anticipated to be a `gdas` job:
```suggestion
        dep_dict = {'type': 'task', 'name': f'{self.run}aeroanlgenb', 'offset': f"-{timedelta_to_HMS(self._base['cycle_interval'])}"}
```

Reply:
so aeroanlinit will run for both gdas and gfs, but aeroanlgenb will only be for gdas. Does that mean that what I have is the correct setup or should it still be `self.run`? Because it would look for a `gfsaeroanlgenb` which does not exist, right?

Reply:
Ah, I see.  Yes, it should stay as it is (`gdasaeroanlgenb`).  Resolving.